packages/@react-spectrum/label/test/StaticField.test.js

```diff
@@ -0,0 +1,255 @@
+    );
+
+    let labelledField = getByTestId('test-id');
+    expect(labelledField).toBeInTheDocument();
+    expect(labelledField).not.toHaveAttribute('aria-describedby');
+  });
+
+  it('renders correctly with RangeValue<CalendarDate>', function () {
+    let {getByTestId} = render(
+      <StaticField
+        data-testid="test-id"
+        label="Field label"
+        value={{start: new CalendarDate(2019, 6, 5), end: new CalendarDate(2019, 7, 5)}} />
+    );
+
+    let labelledField = getByTestId('test-id');
+    expect(labelledField).toBeInTheDocument();
+    expect(labelledField).not.toHaveAttribute('aria-describedby');
+  });
+
+  it('renders correctly with number value', function () {
+    let {getByTestId} = render(
+      <StaticField
+        data-testid="test-id"
+        label="Field label"
+        value={10} />
+    );
+
+    let labelledField = getByTestId('test-id');
+    expect(labelledField).toBeInTheDocument();
+    expect(labelledField).not.toHaveAttribute('aria-describedby');
+  });
+
+  it('renders correctly with RangeValue<NumberValue>', function () {
+    let {getByTestId} = render(
+      <StaticField
+        data-testid="test-id"
+        label="Field label"
+        value={{start: 10, end: 20}} />
+    );
+
+    let labelledField = getByTestId('test-id');
+    expect(labelledField).toBeInTheDocument();
+    expect(labelledField).not.toHaveAttribute('aria-describedby');
+  });
+
+  it('attaches a user provided ref to the outer div', function () {
+    let ref = React.createRef();
+    let {getByTestId} = render(
+      <StaticField
+        data-testid="test-id"
+        ref={ref}
+        value="test" />
+    );
+
+    let labelledField = getByTestId('test-id');
+    expect(ref.current).toBe(labelledField);
+  });
+
+  it('attaches a user provided ref to the outer div with a label', function () {
+    let ref = React.createRef();
+    let {getByTestId} = render(
+      <StaticField
+        data-testid="test-id"
+        label="Field label"
+        ref={ref}
+        value="test" />
+    );
+
+    let labelledField = getByTestId('test-id');
+    expect(ref.current).toBe(labelledField);
+  });
+
+  it('labelPosition: side supports a ref', function () {
+    let ref = React.createRef();
+    let {getByTestId} = render(
+      <StaticField
+        data-testid="test-id"
+        label="Field label"
+        labelPosition="side"
+        ref={ref}
+        value="test" />
+    );
+
+    let labelledField = getByTestId('test-id').closest('div');
+    expect(ref.current).toBe(labelledField);
+  });
+
+  it('renders when no visible label is provided', function () {
+    let {getByTestId} = render(
+      <StaticField
+        data-testid="test-id"
+        value="test" />
+    );
+
+    let labelledField = getByTestId('test-id');
+    expect(labelledField).toBeInTheDocument();
+    expect(labelledField).not.toHaveAttribute('aria-describedby');
```

Comment:
it would be good on each of these to also include the expected formatted text
```
expect(labelledField).toHaveTextContent('test');
```
note toHaveTextContent may not be a valid matcher, might need to do
```
expect(labelledField.textContent).toBe('test');
```